### PR TITLE
brian-algo added.C/C++

### PR DESCRIPTION
**Pull request by MdAkdas**
**Fixes issue:** #4764

**Changes:** New Algorithm

### CODE REVIEW

**Comment by MdAkdas** [comment]
also review this. @arnavb .

File: code/bit_manipulation/src/count_set_bit/brian_Kernighan_Algorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//Brian Kernighan’s Algorithm. This programme uses O(logn) to count set bits.
+//Part of Cosmos by OpenGenus Foundation 
+#include <bits/stdc++.h>
```
Comment (arnavb) [inline code/bit_manipulation/src/count_set_bit/brian_Kernighan_Algorithm.cpp]: #1915. Also move this file to its own directory. Delete the binary file as well.

File: code/bit_manipulation/src/count_set_bit/brian_Kernighan_Algorithm.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
+//Brian Kernighan’s Algorithm. This programme uses O(logn) to count set bits.
+//Part of Cosmos by OpenGenus Foundation 
+#include <bits/stdc++.h> 
+using namespace std;
```
Comment (arnavb) [inline code/bit_manipulation/src/count_set_bit/brian_Kernighan_Algorithm.cpp]: Don't use this.

**Comment by MdAkdas** [comment]
@arnavb please check this.

File: code/bit_manipulation/src/count_set_bits/brian_Kernighan_Algorithm.cpp [hunk 3]
```diff
@@ -0,0 +1,29 @@
+//Brian Kernighan’s Algorithm. This programme uses O(logn) to count set bits.
+//Part of Cosmos by OpenGenus Foundation 
+#include <iostream> 
+using namespace std;
```
Comment (arnavb) [inline code/bit_manipulation/src/count_set_bits/brian_Kernighan_Algorithm.cpp]: Don't use this. Fix your filename to be all lowercase and move the algorithm to its own folder.

File: code/bit_manipulation/src/count_set_bits/brian_kernighan_algo/brian_kernighan_algorithm.cpp [hunk 4]
```diff
@@ -0,0 +1,28 @@
+//Brian Kernighan’s Algorithm. This programme uses O(logn) to count set bits.
+
+#include <iostream> 
+using namespace std;
```
Comment (arnavb) [inline code/bit_manipulation/src/count_set_bits/brian_kernighan_algo/brian_kernighan_algorithm.cpp]: Don't use using namespace std;.

File: code/bit_manipulation/src/count_set_bits/brian_kernighan_algo/brian_kernighan_algorithm.cpp [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+int main() 
+{ 
+	int n;
+	cout << "Enter a positive integer : ";
+	cin >> n;
+	cout << countSetBits(n); 
+
+	return 0; 
+}
```
Comment (arnavb) [inline code/bit_manipulation/src/count_set_bits/brian_kernighan_algo/brian_kernighan_algorithm.cpp]: Add a trailing newline.

Comment (MdAkdas) [reply code/bit_manipulation/src/count_set_bits/brian_kernighan_algo/brian_kernighan_algorithm.cpp]: added.

**Comment by MdAkdas** [comment]
@arnavb @AdiChat  please label this PR so that i can get my points.

**Pull request merged**